client/css/StyleSelector.scss

```diff
@@ -1,18 +1,41 @@
 .stylesAvailable {
-  margin: 10px 0;
-  display: inline-flex;
+  display: flex;
   flex-wrap: wrap;
-  height: auto;
-  align-content: flex-start;
-  width: 100%;
+  padding: 15px 0;
+
+  &__list-item {
+    height: 75px;
+    margin: 5px;
+    position: relative;
+    width: 22%;
+
+    & svg {
+      stroke-width: 20px;
+    }
+  }
+
+  &__selected {
+    position: absolute;
```

Comment:
I finally understand how position relative and position absolute works together now

Reply:
It took me a while as well. My absolute elements use to land all over the place, but it has become a pretty useful tool in the right situation.